controller/adapting_monitor_13.py

```diff
@@ -25,10 +25,21 @@ def __init__(self, *args, **kwargs):
         self.configure(config_file, self.logger)
 
         self.datapaths = {}
-        self.qos_managers: Dict[int, QoSManager] = {}  # Key: datapath id
+        self.qos_manager = QoSManager(AdaptingMonitor13.FLOWS_LIMITS, self.logger)
         self.stats: Dict[int, FlowStatManager] = {}  # Key: datapath id
         self.monitor_thread = hub.spawn(self._monitor)
 
+    def close(self):
+        self.logger.info("adapting_monitor: Deleting queues")
+        # Rules seem to be unset by the time this function is called, therefore unseting them would be wrong.
```

Comment:
WRONG. Rules remain in place and they cause the same problem as before, described in #13.

Reply:
Worked around with `flow_cleaner_13.py`.